src/components/ui-shell/platform-side-nav.hbs

```diff
@@ -1,13 +1,15 @@
 {{#if sidenav.state.expanded}}
-<aside class="bx--side-nav bx--side-nav--expanded">
+  <aside class="bx--side-nav bx--side-nav--expanded">
```

Comment:
We try to not indent with the `if` blocks so that the HTML markup is indented correctly 👍

Reply:
Gotcha. Obviously I'm pretty unfamiliar with Handlebars formatting standards, but we'll get there!